.github/labeler.yml

```diff
@@ -1,6 +1,6 @@
 "c api":
   - changed-files:
-    - any-glob-to-any-file: "src/lib*-c/**/*"
+    - any-glob-to-any-file: "subprojects/lib*-c/**/*"
     - any-glob-to-any-file: "test/unit/**/nix_api_*"
```

Comment:
Is this rename really needed?

Reply:
Oh I see it's discussed in https://github.com/NixOS/nix/pull/10839#issuecomment-2145830083

Reply:
Turns out it is not!